recipes/chakra/index.ts

```diff
@@ -0,0 +1,82 @@
+import {RecipeBuilder, paths, addImport} from "@blitzjs/installer"
+import {builders} from "ast-types/gen/builders"
+import {ASTNode} from "ast-types/lib/types"
+import {NamedTypes} from "ast-types/gen/namedTypes"
+import {visit} from "ast-types"
+
+// Copied from https://github.com/blitz-js/blitz/pull/805, let's add this to the @blitzjs/installer
+function wrapComponentWithCacheProvider(ast: ASTNode, b: builders, t: NamedTypes) {
+  if (!t.File.check(ast)) return
+
+  visit(ast, {
+    visitFunction(path) {
+      // const {node} = path
+      // TODO: need a better way to detect the custom App function
+      // check if is default export
+      if (false) {
+        return this.traverse(path)
+      }
+      return false
+    },
+    visitJSXElement(path) {
+      const {node} = path
+      path.replace(
+        b.jsxElement(
+          b.jsxOpeningElement(b.jsxIdentifier("ThemeProvider")),
+          b.jsxClosingElement(b.jsxIdentifier("CacheProvider")),
+          [
+            b.literal("\n  \t  "),
```

Comment:
hm. shouldn't need to do spacing here, `recast` knows about `prettier` so it should auto-format everything. are you seeing otherwise?

Reply:
I can confirm that auto formatting is not working.

Reply:
oof. i'll look into this. recast should be picking up the prettier config

Reply:
oh. i bet it's because we put the prettier config in `package.json` instead of a separate `.prettier.config.js`